srtcore/core.cpp

```diff
@@ -10712,11 +10712,15 @@ void CUDT::checkRexmitTimer(const steady_clock::time_point& currtime)
     // - LATEREXMIT
     // - flight window == 0
     // - the sender loss list is empty (the receiver didn't send any LOSSREPORT, or LOSSREPORT was lost on track)
-    if (is_laterexmit && unsent_seqno != m_iSndLastAck && m_pSndLossList->getLossLength() == 0)
+    if ((is_laterexmit && unsent_seqno != m_iSndLastAck && m_pSndLossList->getLossLength() == 0)
+    // OF:
+            // - FASTREXMIT
+            // - flight window > 0
+         || (is_fastrexmit && getFlightSpan() != 0))
```

Comment:
Previously the condition was `seqoff(..) > 0`.
`CSeqNo::seqoff(..)` can return negative result.
Now the condition is `CSeqNo::seqlen(..) != 0`.
Looks ok, but still what are your thoughts on the potential changes in behavior?

Reply:
There's no intended change of the behavior. First please tell me what you think on the definition of `getFlightSpan`. Might be that I was kinda confused or had a better idea later - this then should be fixed into `getFlightSpan() + 1 != 0`. This then should be ok because the least value this function can return in this case if -1.

Note that in some places there's used `getFlightSpan()` and in others `getFlightSpan()+1`. I can see now some discrepancies in these statements, but I'm not sure if the definition of `getFlightSpan()` should include this extra -1 or not (in which case it should be added where pure call to this is done and the extra +1 removed).